docs/mendel.md

```diff
@@ -0,0 +1,30 @@
+# nf-core/configs: MENDEL Configuration
+
+All nf-core pipelines have been successfully configured for use on the MENDEL CLUSTER at the Gregor Mendel Institute (GMI).
+
+To use, run the pipeline with `-profile conda,mendel`. This will download and launch the [`mendel.config`](../conf/mendel.config) which has been pre-configured with a setup suitable for the MENDEL cluster. A Conda environment will be created automatically and software dependencies will be downloaded from ['bioconda'](https://bioconda.github.io/).
+
+Theoretically, using `-profile singularity,mendel` would download a docker image containing all of the required software, and convert it to a Singularity image before execution of the pipeline. However, there is a regression in the Singularity deployment on MENDEL which renders containers downloaded from public repositories unusable because they lack the /lustre mountpoint.
```

Comment:
Have you contacted your sysadmins about this? A number of users have encountered the same problem, but it's usually fixable by changing some configuration options with the singularity installation..

Reply:
```
singularity {
  enabled = true
  autoMounts = true
}
```
The `autoMounts` switch worked for us.

Reply:
Would make the docs and config much simpler if you insist that Singularity has to work, or find a solution to the mount problem. Just avoid version 3.0.0!

https://github.com/nextflow-io/nextflow/issues/897

Reply:
Thanks for the suggestions! 
Our compute nodes run a rather dated linux kernel which doesn't support overlayfs, that's why it doesn't work. Atleast that's what sysadmins keep telling me. I'll dig a bit deeper into this